pulp_python/app/models.py

```diff
@@ -63,6 +67,32 @@ def content_handler(self, path):
             name = path.parts[1]
         elif path.match("pypi/*/json"):
             name = path.parts[1]
+        # Ignore the google pub/sub link when running the test scripts (it breaks the docs scripts)
+        if path.match("*.tar.gz") or path.match("*.whl"):
+            try:
+                project_id = settings.GOOGLE_PUBSUB_PROJECT_ID
+                topic_id = settings.GOOGLE_PUBSUB_TOPIC_ID
+
+                publisher = pubsub_v1.PublisherClient()
+                topic_path = publisher.topic_path(project_id, topic_id)
+
+                message = json.dumps({
+                    "action": "package_requested",
+                    "package": str(path),
+                    "source": self.base_path,
+                })
+
+                response = publisher.publish(topic_path, message.encode("utf-8"))
+                log.info(
+                    "package_requested message send to %s pub/sub, %s",
+                    topic_id,
+                    response.result()
+                )
+            except Exception as e:
+                log.error(
+                    "Could not call package_requested message to pub/sub server, %s",
+                    e.__str__()
+                )
```

Comment:
```suggestion
            except Exception:
                log.exception(
                    "Could not call package_requested message to pub/sub server"
                )
```

this is the [more idiomatic way](https://docs.python.org/3/library/logging.html#logging.Logger.exception) of achieving the same.